docs/src/content/docs/api/calls.mdx

```diff
@@ -99,19 +99,25 @@ Lists all calls that have been created. API key scoped.
 ## Create Call
 <Badge text="POST /calls" variant="success" size="large" />
 
-Creates a new call using the specified system prompt. Optional parameters can be used to specify voice, temperature, and lanugage hint. API key scoped.
+Creates a new call using the specified system prompt. API key scoped.
+
+Optional parameters can be used to specify voice, temperature, language hint, and recordingEnabled.
+
+An optional query parameter called `priorCallId` can be provided to continue an existing call (AKA resume a call).
```

Comment:
```suggestion
An optional query parameter called `priorCallId` can be provided to continue a previous conversation.
```

This **does not** continue an existing call (which would make billing challenging). It still creates a new call, just using all the optional properties and message history from the prior call unless overridden.